src/xenia/kernel/xam/xam_user.cc

```diff
@@ -991,14 +975,19 @@ dword_result_t XamUserGetOnlineCountryFromXUID_entry(qword_t xuid) {
 DECLARE_XAM_EXPORT1(XamUserGetOnlineCountryFromXUID, kUserProfiles,
                     kImplemented);
 
-constexpr uint8_t kStatsMaxAmount = 64;
-
-struct X_STATS_DETAILS {
-  xe::be<uint32_t> id;
-  xe::be<uint32_t> stats_amount;
-  xe::be<uint16_t> stats[kStatsMaxAmount];
-};
-static_assert_size(X_STATS_DETAILS, 8 + kStatsMaxAmount * 2);
+dword_result_t XamUserIsParentalControlled_entry(dword_t user_index) {
+  /* Notes:
+      - if (data_address < 1 || XamExecutingOnBehalfOfTitle == 0 || title id ==
+     kDashboardID || user_type != offline) (type mask used in XamUserGetXUID) go
+     forward else return false
+  */
+  const auto& user = kernel_state()->xam_state()->GetUserProfile(user_index);
+  if (!user) {
+    return false;
+  }
+  return user->IsParentalControlled();
```

Comment:
I'm kinda worried that for some custom profiles or profiles from console it might limit something, so might be wise to just return that it isn't controlled